_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -222,26 +392,220 @@ A set of two SVG `<a>` elements have the same [accessible name][] but link to di
 
 #### Failed Example 6
 
-Links resolves to same resource after redirect, but the redirect is not instant:
+Both links resolve to [same resource][] after redirect, but the redirect is not instant.
 
 ```html
 <a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html">Contact us</a>
 <a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/redirect1.html">Contact us</a>
 ```
 
+#### Failed Example 7
+
+Both links have the same [accessible name][]. When the [shadow tree][] is attached to `host` and flattened, both links appear in the [flat tree][] (work in progress). Since they have the same [accessible name][] but point to different resources, the rule fails.
+
+```html
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Links in the shadow</title>
+	</head>
+
+	<body>
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Contact us</a>
+		all the time.
+
+		<div id="host"></div>
+
+		<script>
+			const host = document.getElementById('host')
+			const shadowRoot = host.attachShadow({ mode: 'open' })
+
+			shadowRoot.innerHTML =
+				'<span><a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html">Contact us</a> from the shadow.</span>'
+		</script>
+	</body>
+</html>
+```
+
+#### Failed Example 8
```

Comment:
See my comment on passed example 12. This rule isn't about correct use of shadow DOM, so it's test shouldn't be either. Failed examples 8 through 11 should be removed.